.github/workflows/nightly.yaml

```diff
@@ -3,6 +3,8 @@ name: Nightly Releases
 on:
   push:
     branches: [ dev, ci/automatic-building ]
+  pull_request:
```

Comment:
I'd be inclined to keep things simple and just address the spaces issue in this PR, so nightly builds carry on working as before. Whether we really want to do a CI build for every PR warrants a bit of thought I think.

Reply:
I initially had it removed because I was not convinced 100% that it was necessary. I think, given the future may have more testing involved that we could have tests run on a PR in a similar fashion, as well as a build to see if it will work properly. A question mark to be resolved I reckon but for now let's see if it is useful?

Reply:
If you like; I guess it seems profligate in terms of launching lots and lots of builds (I can obviously see the utility though). In any case, not in the same merge commit as the fix for this issue would be preferable from a history-hygiene POV, IMO, YMMV, etc.

Reply:
Yeah okay. I'll cherry wangle this mofo into another one.

Reply:
❤️

Reply:
#97